src/encoded/schemas/experiment_type.json

```diff
@@ -0,0 +1,151 @@
+{
+    "title": "Experiment Type",
+    "description": "Schema for experiment type",
+    "id": "/profiles/experiment_type.json",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "type": "object",
+    "required": ["title"],
+    "identifyingProperties": ["experiment_name", "uuid", "aliases"],
+    "additionalProperties": false,
+    "mixinProperties": [
+        { "$ref": "mixins.json#/schema_version" },
+        { "$ref": "mixins.json#/uuid" },
+        { "$ref": "mixins.json#/submitted" },
+        { "$ref": "mixins.json#/modified" },
+        { "$ref": "mixins.json#/attribution" },
+        { "$ref": "mixins.json#/static_embeds" },
+        { "$ref": "mixins.json#/notes" }
+    ],
+    "properties": {
+        "schema_version": {
+            "default": "1"
+        },
+        "lab": {
+            "default": "828cd4fe-ebb0-4b36-a94a-d2e3a36cc989",
+            "permission": "import_items"
+        },
+        "award": {
+            "default": "b0b9c607-f8b4-4f02-93f4-9895b461334b",
+            "permission": "import_items"
+        },
+        "status": {
+            "title": "Status",
+            "type": "string",
+            "default": "in review by lab",
+            "enum" : [
+                "released",
+                "in review by lab",
+                "deleted"
+            ]
+        },
+        "title": {
+            "title": "Experiment Type",
+            "description": "The display name for the experiment type",
+            "type": "string",
+            "lookup": 20
+        },
+        "experiment_name": {
+            "description": "DON'T SUBMIT, auto-generated",
+            "type": "string",
+            "pattern": "^[a-z0-9\\-]+$",
+            "uniqueKey": true,
+            "exclude_from": ["submit4dn", "FFedit-create"]
+        },
+        "controlled_term": {
+            "title": "Ontology Term",
+            "description": "Ontology term that matches this experiment type",
+            "type": "string",
+            "linkTo": "OntologyTerm"
+        },
+        "experiment_category": {
+            "title": "Experiment Category",
+            "description": "General category of the experiment type",
+            "type": "string",
+            "enum": ["Sequencing", "Microscopy"],
+            "lookup": 100
+        },
+        "assay_classification": {
```

Comment:
For assay_classification and subclassification what about making them suggested_enums?  Makes it significantly easier to add/change the values.